server/journeys/screen/confirmation/view.njk

```diff
@@ -0,0 +1,19 @@
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% extends "partials/layout.njk" %}
+
+{% set pageTitle = "Confirmation - Screen CSIP referral" %}
+
+{% set theHtml = ("<div><strong>Decision: </strong>" + outcomeTypeDescription + "</div>") %}
```

Comment:
`outcomeTypeDescription | escape`

Reply:
Shouldn't need to escape here because outcomeTypeDescription is only populated by reference data - but it doesn't hurt to add that